src/components/checkbox/checkbox.html

```diff
@@ -11,7 +11,8 @@
            [attr.aria-labelledby]="ariaLabelledby"
            (focus)="onInputFocus()"
            (blur)="onInputBlur()"
-           (change)="onInteractionEvent($event)">
+           (change)="onInteractionEvent($event)"
+           (click)="onInteractionEvent($event)">
```

Comment:
upon clicking, checkbox fires click function twice. Ideally it should fire it once right?